From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Reilly Brogan <reilly@reillybrogan.com>
Date: Tue, 26 Aug 2025 12:53:54 -0500
Subject: [PATCH] downstream: Update default PATH

---
 src/env/environment.rs | 5 +++--
 1 file changed, 3 insertions(+), 2 deletions(-)

diff --git a/src/env/environment.rs b/src/env/environment.rs
index ce70b9b44..07a0917c8 100644
--- a/src/env/environment.rs
+++ b/src/env/environment.rs
@@ -543,9 +543,10 @@ fn setup_user(global_exported_mode: EnvSetMode, vars: &EnvStack) {
         colon_split(&[cstr2wcstring(cstr)])
     } else {
         vec![
-            str2wcstring(PREFIX) + L!("/bin"),
+            L!("/usr/local/bin").to_owned(),
+            L!("/usr/local/sbin").to_owned(),
             L!("/usr/bin").to_owned(),
-            L!("/bin").to_owned(),
+            L!("/usr/sbin").to_owned(),
         ]
     };
     Box::leak(paths.into_boxed_slice())
