diff --git a/main.c b/main.c
index 07e6667..69e5b81 100644
--- a/main.c
+++ b/main.c
@@ -961,6 +961,7 @@ static char *get_config_path(void) {
 		"$HOME/.swaylock/config",
 		"$XDG_CONFIG_HOME/swaylock/config",
 		SYSCONFDIR "/swaylock/config",
+		"/usr/share/defaults/swaylock/config",
 	};
 
 	char *config_home = getenv("XDG_CONFIG_HOME");
diff --git a/meson.build b/meson.build
index 1a944ee..e9694f5 100644
--- a/meson.build
+++ b/meson.build
@@ -129,7 +129,7 @@ executable('swaylock',
 if libpam.found()
 	install_data(
 		'pam/swaylock',
-		install_dir: get_option('sysconfdir') / 'pam.d'
+		install_dir: get_option('prefix') / 'share/defaults/pam.d'
 	)
 endif
 
